holoviews/util.py

```diff
@@ -28,14 +28,17 @@ class Dynamic(param.ParameterizedFunction):
     kwargs = param.Dict(default={}, doc="""
         Keyword arguments passed to the function.""")
 
+    shared_data = param.Boolean(default=False, doc="""
+        Whether the cloned DynamicMap will share the same data.""")
```

Comment:
Ah yes, thanks! I suppose you might want to copy the cache in some cases (e.g for ``relabel``).